ims/einstein/operations.py

```diff
@@ -681,3 +682,26 @@ def run_script(self, img, script):
                                    constants.DEFAULT_SNAPSHOT_NAME)
             self.fs.remove_snapshot(ceph_img_name,
                                     constants.DEFAULT_SNAPSHOT_NAME)
+
+    # TODO: Write tests for this method before merging to main branch
+    @log
+    def get_iscsi_target(self, node_name):
+        try:
+            if not self.is_admin:
+                raise AuthorizationFailedException()
+            provisioned = False
+            provisioned_nodes = self.list_provisioned_nodes()['retval']
+            for node in provisioned_nodes:
+                if node_name == node[0]:
+                    provisioned = True
+            if not provisioned:
+                raise iscsi_exceptions.TargetDoesntExistException()
+            file_path = "/var/lib/tftpboot/" + node_name + ".ipxe"
```

Comment:
This could be separated into another fn and should be in dnsmasq.py I guess.(this class name should be changed to PXEServer). Refer #75